crates/admin/src/cluster_controller/service.rs

```diff
@@ -280,12 +280,14 @@ impl<T: TransportConnect> Service<T> {
         let mut partition_table_watcher = self.metadata.watch(MetadataKind::PartitionTable);
         let mut logs = self.metadata.updateable_logs_metadata();
         let mut partition_table = self.metadata.updateable_partition_table();
+        let mut tail_udpates_watcher = logs_controller.watch_logs_tail_updates();
 
         loop {
             tokio::select! {
                 _ = self.heartbeat_interval.tick() => {
                     // Ignore error if system is shutting down
                     let _ = self.cluster_state_refresher.schedule_refresh();
+                    let _ = logs_controller.schedule_refresh_tails();
```

Comment:
How expensive is it to perform this - does it need its own frequency decoupled from cluster node heartbeating?

Reply:
Yeah, i wasn't sure if we should either 
- Make it less frequent (different timer) 
- Only trigger if an external reconfiguration detected (NewLogs event)

Reply:
Would it make more sense if we schedule refresh tail iff NewLogs event was received? @pcholakov and @tillrohrmann

Reply:
Only reacting to the `NewLogs` event won't be enough, I fear. Another `LogsController` or your future self might have gotten to the point where it started the sealing procedure but did not update the `Logs` yet.

Reply:
I would agree with Pavel that the heartbeat interval is probably a bit too often. Right now, the problem of a half-sealed loglet can happen if there is a concurrent `LogsController` that fails updating the `Logs` configuration (e.g. it crashed or there was a concurrent `Logs` update). For the latter case, we can indeed react to `NewLogs` events (and later on leadership changes of the CC) to trigger the check. Having this, we only need a safety-net for a concurrent CC that was sealing and then crashed w/o getting restarted. Here, maybe a check every 5 minutes is probably good enough to get the cluster out of a stuck state. In the future, we also might monitor the progress of PPs for an additional signal to check the tail status.